### PR TITLE
Change PR template to include a note about image versions changes

### PR DESCRIPTION
**Pull request by karan**
**What this PR does / why we need it**: Change PR template to include a note about image versions changes, per SIG meeting today.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

@kubernetes/kube-deploy-reviewers

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -20,6 +23,3 @@ Fixes #
 ```release-note
 
 ```
-
```
Comment (k4leung4) [inline .github/PULL_REQUEST_TEMPLATE.md]: should we replace this with the new reviewer/maintainer group?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,6 +13,8 @@ Fixes #
 
 **Special notes for your reviewer**:
 
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
```
Comment (mkjelland) [inline .github/PULL_REQUEST_TEMPLATE.md]: Aren't there cases where a PR could break functionality unless an image is updated? I have seen this happen a couple times already.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (roberthbailey) [reply .github/PULL_REQUEST_TEMPLATE.md]: I don't see aren't any groups in https://github.com/orgs/kubernetes-sigs/people (compare with https://github.com/orgs/kubernetes/people).

**Comment by roberthbailey** [comment]
lgtm

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (k4leung4) [reply .github/PULL_REQUEST_TEMPLATE.md]: When we get integration tests, would this complicate things? as in the test will run at HEAD with older images.

**Pull request merged**